sound/soc/intel/boards/sof_sdw.c

```diff
@@ -671,6 +680,20 @@ static struct sof_sdw_codec_info codec_info_list[] = {
 		.init = sof_sdw_rt5682_init,
 		.codec_type = SOF_SDW_CODEC_TYPE_JACK,
 	},
+	{
+		.part_id = 0x8363,
+		.direction = {true, false},
+		.dai_name = "max98363-aif1",
+		.init = sof_sdw_mx8373_init,
+		.codec_type = SOF_SDW_CODEC_TYPE_AMP,
+	},
+	{
+		.part_id = 0x4242,
+		.direction = {true, true},
+		.dai_name = "cs42l42-sdw",
+		.init = sof_sdw_cs42l42_init,
+		.codec_type = SOF_SDW_CODEC_TYPE_JACK,
+	},
```

Comment:
this is going to conflict with your PR @bardliao

Reply:
> this is going to conflict with your PR @bardliao

@plbossart Can we merge https://github.com/thesofproject/linux/pull/4191 now? I also need to add rt712 and Cirrus Logic codecs support to the machine driver.

Reply:
done @bardliao